src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,23 @@
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import { color, typeface } from '../../styles';
+
+const style = css`
+  display: flex;
+  align-items: center;
+  padding: 0 1rem;
+  font-size: 1rem;
+  * {
+    margin: 0 1rem;
+  }
+`;
+
+interface Props {
```

Comment:
can you add a Prop to pass for `backgroundColor` to the navbar? I'm fairly sure we'll want to have something like that eventually - we've got at least 3 flavors of nav on the site at the moment:

![image](https://user-images.githubusercontent.com/1844496/65422561-d2be4180-ddd4-11e9-96c1-c5c4d1a96bb9.png)


And what do you think about a mechanism to send content to the right side of nav, like above?

Reply:
@mbifulco 
Good morning Mike, 
I can make the alignment work with an additional parameter.
For the background color, my thought was to let it inherit the background of its parent container for full flexibility. That being said, I can make the change to set the background colour. 
Let me know your preference.

Reply:
Hmm - I still don't think what's here is right.  How would we recreate the middle bar from the screenshot above? `Gymnasium` Logo on the left, and links on the right?

Reply:
@mbifulco OK, I will add that.